audit/projects/k8s-staging-e2e-test-images/buckets/artifacts.k8s-staging-e2e-test-images.appspot.com/iam.json

```diff
@@ -4,7 +4,8 @@
       "members": [
         "group:k8s-infra-artifact-admins@kubernetes.io",
         "projectEditor:k8s-staging-e2e-test-images",
-        "projectOwner:k8s-staging-e2e-test-images"
+        "projectOwner:k8s-staging-e2e-test-images",
+        "serviceAccount:456067983721@cloudbuild.gserviceaccount.com"
```

Comment:
I don't think this belongs here, but I'll allow it for now.

Would like to revisit GCR and GCB permissions in general based on the auditing I was doing while troubleshooting in https://github.com/kubernetes/test-infra/issues/20884#issuecomment-783712871